src/features/purchases/screen/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { SafeAreaView, SectionList, Text, View } from 'react-native';
+
+import { translate } from 'localization/hooks';
+
+import { useGetOrders } from 'network/queries/order-queries';
+
+import OrderItem from './orderItem';
+import styles from './styles';
+
+const PurchasesScreen: React.FunctionComponent = () => {
+  const { data } = useGetOrders();
+  const orders = data?.data || [];
```

Comment:
In the useQuery you can just return:
`{orders: data?.data || [] }`

That way, components are cleaner and more readable.